class-wc-settings-page-komoju.php

```diff
@@ -62,6 +56,18 @@ public function get_settings($current_section = '')
         return apply_filters('woocommerce_get_settings_' . $this->id, $settings, $current_section);
     }
 
+    // Override from WC_Settings_Page
+    public function save()
+    {
+        $old_payment_types = get_option('komoju_woocommerce_payment_types');
+        parent::save();
+        $new_payment_types = get_option('komoju_woocommerce_payment_types');
+
+        if ($old_payment_types != $new_payment_types) {
+            $this->cache_payment_methods_from_komoju($old_payment_types, $new_payment_types);
+        }
+    }
```

Comment:
I replaced the `add_action` above with this `save` override to fix a bug that I uncovered during testing...

Basically, there are some circumstances where this class is not yet initialized at the time that `update_option` gets called on `'komoju_woocommerce_payment_types'`. Using `save` like this makes absolutely sure that our cache from komoju is correct!